### PR TITLE
if metro >= 0.51, require from metro-react-native-babel-transformer

### PR DESCRIPTION
**Pull request by smeevil**
This solves: https://github.com/bamlab/react-native-graphql-transformer/issues/6

### CODE REVIEW

**Comment by sterlingwes** [comment]
@tpucci can we have this ❓ 🙏

File: index.js [hunk 1]
```diff
@@ -5,10 +5,14 @@ let upstreamTransformer = null;
 
 const reactNativeVersionString = require('react-native/package.json').version;
 
+const metroVersion = parseFloat(require('metro/package.json').version);
```
Comment (tpucci) [inline index.js:8]: Parsing the version as a float seems a little hacky to me.
Could you use the same syntax as `reactNativeMinorVersion` ?

Like:
```js
const metroVersionString = require('metro/package.json').version;

const metroMinorVersion = semver(metroVersionString).minor;
```

File: index.js [hunk 2]
```diff
@@ -5,10 +5,14 @@ let upstreamTransformer = null;
...
+
 const reactNativeMinorVersion = semver(reactNativeVersionString).minor;
 
-if (reactNativeMinorVersion >= 56) {
-  upstreamTransformer = require('metro/src/reactNativeTransformer')
+if (metroVersion >= 0.51) {
```
Comment (tpucci) [inline index.js:12]: Then, this would become: 

```js
if (metroMinorVersion >= 51) {
...
}
```

**Review by tpucci — CHANGES_REQUESTED** [review]
I think we should use semver to compare versions. Parsing as a float may be dangerous if metro ever decide to use the semver standard someday.

**Comment by duyleekun** [comment]
Anyone?

**Pull request merged**